openshift/authorize/networkSecurityPolicy/allNamespacesK8s.yml

```diff
@@ -0,0 +1,42 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+
+parameters:
+  - description: airflow_prefix
+    displayName: airflow_prefix
+    name: AIRFLOW_PREFIX
+    required: true
+  - description: ggircs_prefix
+    displayName: ggircs_prefix
+    name: GGIRCS_PREFIX
+    required: true
+  - description: ciip_prefix
+    displayName: ciip_prefix
+    name: CIIP_PREFIX
+    required: true
+
+objects:
+  - kind: NetworkSecurityPolicy
+    apiVersion: security.devops.gov.bc.ca/v1alpha1
+    metadata:
+    name: custom-all-namespaces-k8s-api-communication
```

Comment:
If we're creating NSPs for service-accounts specific to a deployment (ie: airflow-worker), is it still at 'base' NSP? Or should templates be added to the actual deployments to create them? (I can still add the 'builder' / 'deployer' ones as base NSPs)

Reply:
If we're having airflow spin up job pods in namespaces that are not airflow (ie deploy_ciip_db in $CIIP_NAMESPACE), then won't that $CIIP_NAMESPACE also need a connection to the k8s api? Since the job pods will be running in that namespace?

Reply:
Nevermind, all the API calls are in the dags

Reply:
The builder role is obsolete as we are now building on CI. I think that we might want to create a `job-runner` role, and give it to the airflow service account?

Reply:
so instead of binding the destination to a service account name we would define a `job-runner` role and use the label -role=job-runner in the NSP?